modules/apps/segments/segments-web/src/main/resources/META-INF/resources/js/components/segment_edit/SegmentEdit.es.js

```diff
@@ -42,7 +47,54 @@ class SegmentEdit extends Component {
 	};
 
 	state = {
-		membersCount: this.props.initialMembersCount || 0
+		membersCount: this.props.initialMembersCount || 0,
```

Comment:
Would it be helpful to use https://reactjs.org/docs/react-component.html#static-getderivedstatefromprops instead?

Reply:
Since `getDerivedStateFromProps` also gets called on subsequent updates, I don't think it would be applicable for this case since `initialMembersCount` should only be copied on the initial mount.